### ISSUE TITLE
[DATA CONTRIBUTION] Caravan extension Israel

### ISSUE THREAD

**Issue by EfratMorin** [kratzert/Caravan#18]
### Basin prefix

il

### Zenodo DOI

https://doi.org/10.5281/zenodo.7758516

### Number of catchments

95

### Location of catchments

Israel

### For which periods are streamflow records available in your dataset?

01/01/1970-01/01/2021

### Please list any sources of the data you contributed.

https://data.gov.il/dataset/level_discharge
https://www.gov.il/he/departments/topics/water_authority_maps/govil-landing-page

### License

CC-BY-4.0

### Checklist

- [X] I have uploaded my dataset on Zenodo, where it is accessible under the DOI provided above.
- [X] I used a basin prefix that is not yet used by any other Caravan sub-dataset (you can check this via the Data Contributions discussion thread (TODO link), where all accepted Caravan contributions are listed).
- [X] Permissive License: My data is available under a license that is compatible with the Caravan CC-BY-4.0 license (the easiest way to be sure about this is if your data uses CC-BY-4.0, too).

**Comment by kratzert** [comment]
Hi Efrat, 

I just checked everything and everything looks good to me. Thank you so much for contributing to Caravan and for adding ~100 gauges in Israel to Caravan. 

One note though:
- There is one basin in the dataset with 4 attributes missing. The basin is `il_25191` and the attributes are all related to soil. I manually checked the underlying HydroATLAS polygons and indeed, they contain the "NoData" value, so there is nothing we can do about it.

I just wanted to add this here so we are aware of it.

There is nothing more to do from your side. I will close this issue and add your contribution to the [Community Extension Overview](https://github.com/kratzert/Caravan/discussions/10) thread.

Again, thank you very much!
